Lecture-14/src/main/java/factory/RestAssuredResponseFactory.java

```diff
@@ -0,0 +1,40 @@
+package factory;
+
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.response.Response;
+import com.jayway.restassured.specification.RequestSpecification;
+import constant.HttpMethodType;
+
+import static com.jayway.restassured.RestAssured.given;
+
+public class RestAssuredResponseFactory {
+
+    private static final String TOKEN = "9831d26fead1dcb4cd9419a9107a84555742f0a3";
+    private static RequestSpecification requestSpecification;
+
+    private RestAssuredResponseFactory() {
+    }
+
+    private static void setup() {
```

Comment:
Move to constructor since it's required for normal class operation

Reply:
done